presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/query/TestPinotQueryGenerator.java

```diff
@@ -366,6 +367,8 @@ public void testSimpleSelectWithTopN()
                 "SELECT regionId, city, fare FROM realtimeOnly ORDER BY fare, city DESC LIMIT 50",
                 defaultSessionHolder,
                 ImmutableMap.of());
```

Comment:
Any idea whether this issue also impacts the Pinot-SQL query generation logic added by @fx19880617 recently ? Perhaps we need to fix it and add a test for this there as well ?

Reply:
For order by clause SQL generation also follows the same path via `generatePinotQueryHelper` so this logic gets added to SQL generation too. 

Good catch on test, I missed the new `TestPinotQueryGeneratorSql` file. Added a test there too